vpr/src/base/netlist_writer.cpp

```diff
@@ -92,6 +92,8 @@
 //File local type declarations
 //
 
+typedef std::pair<double, std::string> sequential_port_delay_pair;
```

Comment:
Please comment what is stored in this type. (what is the string, what is the double, what is it used for?)

Reply:
Comment added